src/main/java/uk/gov/companieshouse/items/orders/api/controller/CertificateItemsController.java

```diff
@@ -70,4 +65,18 @@ public CertificateItemsController(final CreateItemRequestValidator validator,
         return ResponseEntity.status(HttpStatus.CREATED).body(createdCertificateItemDTO);
     }
 
+    @GetMapping("/certificates/{certificate_id}")
```

Comment:
Do we not need to use the configured value `${uk.gov.companieshouse.items.orders.api.path}` here?

Reply:
Oh yes, I will change this now